README.md

```diff
@@ -1,53 +1,129 @@
 # Discord bot
 
-# !!! Currently developing version 2.0 with rewritting code to C# !!!
+[![Docker image build](https://github.com/seker212/Discord_bot/actions/workflows/container-image-update.yml/badge.svg?branch=master-2.0)](https://github.com/seker212/Discord_bot/actions/workflows/container-image-update.yml)
 
-[![Chi-chan bot Container](https://github.com/seker212/Discord_bot/actions/workflows/Chi-chan_Container.yml/badge.svg?branch=master&event=push)](https://github.com/seker212/Discord_bot/actions/workflows/Chi-chan_Container.yml)
+## Table of contents
 
-## Running locally
+1. [Running bot](#tc1)
+    1. [Important informations](#tc11)
```

Comment:
In English there is no plural form with `s` for "information" https://dictionary.cambridge.org/grammar/british-grammar/information